tests/wpt/web-platform-tests/html/webappapis/scripting/processing-model-2/unhandled-promise-rejections/promise-rejection-events-onerror.html

```diff
@@ -15,7 +15,7 @@
 
   window.onerror = function (msg, url, line, col, error) {
     t.step(function() {
-      assert_equals(msg, 'Uncaught Error: e2');
+      assert_equals(msg, 'e2');
```

Comment:
This should assert that `e2` is in the msg string. That should allow Blink to continue passing this test.

Reply:
It must be a better solution! Let me update it, thanks!